src/shogun/features/DenseFeatures.cpp

```diff
@@ -468,21 +438,6 @@ template<class ST> void CDenseFeatures<ST>::initialize_cache()
 
 template<class ST> EFeatureClass CDenseFeatures<ST>::get_feature_class() const  { return C_DENSE; }
 
-template<class ST> bool CDenseFeatures<ST>::reshape(int32_t p_num_features, int32_t p_num_vectors)
```

Comment:
this should just be in the matrix itself, i.e. create a new feature instance using the reshaped sgmatrix